controls/C-0256-exposuretointernet.json

```diff
@@ -20,10 +20,10 @@
             }
         ]
     },
-    "description": "",
-    "remediation": "",
-    "rulesNames": [],
-    "test": "",
+    "description": "fails if workloads are exposed on internet",
+    "remediation": "There's no remediation for this control. This is used to inform the user that is exposing workloads on Internet.",
```

Comment:
We phrase the text to the user (i.e. not "third body"). Also, there is a remediation - the user can evaluate it's resources and apply relevant changes wherever needed.